simanneal/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "simanneal" %}
+{% set version = "0.4.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0657354bacd503ff5fc0b984110be7b174ecedbb976caf1c230c73a128a06d5a
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
```

Comment:
I believe this recipe can add the `noarch: python` directive to the build section.